src/contact.ts

```diff
@@ -153,6 +176,58 @@ export class Contact implements Sayable {
   }
 
   /**
+   * Check if it's a offical account
+   *
+   * @returns {boolean|null} True for official account, Flase for contact is not a official account
+   *
+   * @example
+   * ```ts
+   * const isOfficial = contact.official()
+   * ```
+   */
+  public official(): boolean {
+    return !!this.obj && this.obj.official
+  }
+
+  /**
+   * Check if it's a special contact
+   *
+   * the contact who's id in following list will be identify as a special contact
+   *
+   * ```ts
+   * 'weibo', 'qqmail', 'fmessage', 'tmessage', 'qmessage', 'qqsync', 'floatbottle',
+   * 'lbsapp', 'shakeapp', 'medianote', 'qqfriend', 'readerapp', 'blogapp', 'facebookapp',
+   * 'masssendapp', 'meishiapp', 'feedsapp', 'voip', 'blogappweixin', 'weixin', 'brandsessionholder',
+   * 'weixinreminder', 'wxid_novlwrv3lqwv11', 'gh_22b87fa7cb3c', 'officialaccounts', 'notification_messages',
+   * ```
+   * @see https://github.com/Chatie/wechaty/blob/master/doc/webwxapp.js#L4057
```

Comment:
Please change refer to https://github.com/Chatie/webwx-app-tracker with the exactly commit hash.